aleph/plugins/peinfo.py

```diff
@@ -9,91 +9,10 @@ class PEInfoPlugin(PluginBase):
-            if pe.is_driver():
-                self.sample.add_tag('driver')
-
-            # Compilation time
-            timestamp = pe.FILE_HEADER.TimeDateStamp
-
-            if timestamp == 0:
-                self.sample.add_tag('no-timestamp')
-            else:
-                data['compilation_timestamp'] = timestamp
-                data['compilation_date'] = datetime.datetime.utcfromtimestamp(int(timestamp)).strftime('%Y-%m-%d %H:%M:%S')
-                if (timestamp < 946692000):
-                    self.sample.add_tag('old-timestamp')
-                elif (timestamp > time.time()):
-                    self.sample.add_tag('future-timestamp')
-
-            #data['entry_point'] = pe.OPTIONAL_HEADER.AddressOfEntryPoint
-            #data['image_base']  = pe.OPTIONAL_HEADER.ImageBase
-            data['number_sections'] = pe.FILE_HEADER.NumberOfSections
-
-            #check for ASLR, DEP/NX and SEH
-            if pe.OPTIONAL_HEADER.DllCharacteristics > 0:
-                if pe.OPTIONAL_HEADER.DllCharacteristics & 0x0040:
-                    data['aslr'] = True
-                if pe.OPTIONAL_HEADER.DllCharacteristics & 0x0100:
-                    data['dep'] = True
-                if (pe.OPTIONAL_HEADER.DllCharacteristics & 0x0400
-                or (hasattr(pe, "DIRECTORY_ENTRY_LOAD_CONFIG") 
-                and pe.DIRECTORY_ENTRY_LOAD_CONFIG.struct.SEHandlerCount > 0 
-                and pe.DIRECTORY_ENTRY_LOAD_CONFIG.struct.SEHandlerTable != 0) 
-                or pe.FILE_HEADER.Machine == 0x8664):
-                    data['seh'] = True
-
-            # Check imports
-            if hasattr(pe, 'DIRECTORY_ENTRY_IMPORT'):
-                imports = {}
-                for lib in pe.DIRECTORY_ENTRY_IMPORT:
-                    imports[lib.dll] = []
-                    for imp in lib.imports:
-                        if (imp.name != None) and (imp.name != ""):
-                            imports[lib.dll].append({'address': hex(imp.address), 'name': imp.name})
-
-                data['imports'] = imports
-
-            # Check exports
-            if hasattr(pe, 'DIRECTORY_ENTRY_EXPORT'):
-                exports = []
-                for exp in pe.DIRECTORY_ENTRY_EXPORT.symbols:
-                    exports.append({'address': hex(pe.OPTIONAL_HEADER.ImageBase + exp.address), 'name': exp.name, 'ordinal': exp.ordinal})
-                    if exp.name == 'CPlApplet' and pe.is_dll():
-                        self.sample.add_tag('cpl')
-
-                data['exports'] = exports
-
-            # Get sections
-            if len(pe.sections) > 0:
-                data['sections'] = []
-                for section in pe.sections:
-                    data['sections'].append({'name': section.Name.replace('\x00', ''), 'address': hex(section.VirtualAddress), 'virtual_size': hex(section.Misc_VirtualSize), 'raw_size': section.SizeOfRawData })
-            
-            return data
+            return self.get_pe_data()
```

Comment:
I just create a more verbose function name.
```
What process does? I don't know 
```
But get_pe_data is more intuitive

```
What  get_pe_data does? get pe data
```
I don't change name of process function because there are no tests yet.